README.md

```diff
@@ -8,6 +21,53 @@ high fantasy theme, featuring both singleplayer and online/hotseat multiplayer
 combat. Fight a desperate battle to reclaim the throne of Wesnoth, or take
 hand in any number of other adventures.
 
+Contributing
+-------------
+Interested in creating your content? One of Wesnoth's best features is its extensibility. Players can create new maps, units, races, scenarios, art, music, and even entire campaigns! Please read information on our site (https://wiki.wesnoth.org/Create) for more information.
```

Comment:
> Please read information on our site (https://wiki.wesnoth.org/Create) for more information.

Please change this to "To get started, see the [Create](https://wiki.wesnoth.org/Create) page on the wiki."